specification/guestconfiguration/GuestConfiguration.Management/models.tsp

```diff
@@ -0,0 +1,801 @@
+   */
+  properties?: GuestConfigurationAssignmentReportProperties;
+}
+
+/**
+ * Report for the guest configuration assignment. Report contains information such as compliance status, reason, and more.
+ */
+model GuestConfigurationAssignmentReportProperties {
+  /**
+   * A value indicating compliance status of the machine for the assigned guest configuration.
+   */
+  @visibility(Lifecycle.Read)
+  complianceStatus?: ComplianceStatus;
+
+  /**
+   * GUID that identifies the guest configuration assignment report under a subscription, resource group.
+   */
+  @visibility(Lifecycle.Read)
+  reportId?: string;
+
+  /**
+   * Configuration details of the guest configuration assignment.
+   */
+  assignment?: AssignmentInfo;
+
+  /**
+   * Information about the VM.
+   */
+  vm?: VMInfo;
+
+  /**
+   * Start date and time of the guest configuration assignment compliance status check.
+   */
+  @visibility(Lifecycle.Read)
+  startTime?: utcDateTime;
+
+  /**
+   * End date and time of the guest configuration assignment compliance status check.
+   */
+  @visibility(Lifecycle.Read)
+  endTime?: utcDateTime;
+
+  /**
+   * Details of the assignment report.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  details?: AssignmentReportDetails | null;
+
+  /**
+   * Azure resource Id of the VMSS.
+   */
+  @visibility(Lifecycle.Read)
+  vmssResourceId?: string;
+}
+
+/**
+ * Details of the guest configuration assignment report.
+ */
+model AssignmentReportDetails {
+  /**
+   * A value indicating compliance status of the machine for the assigned guest configuration.
+   */
+  @visibility(Lifecycle.Read)
+  complianceStatus?: ComplianceStatus;
+
+  /**
+   * Start date and time of the guest configuration assignment compliance status check.
+   */
+  @visibility(Lifecycle.Read)
+  startTime?: utcDateTime;
+
+  /**
+   * End date and time of the guest configuration assignment compliance status check.
+   */
+  @visibility(Lifecycle.Read)
+  endTime?: utcDateTime;
+
+  /**
+   * GUID of the report.
+   */
+  @visibility(Lifecycle.Read)
+  jobId?: string;
+
+  /**
+   * Type of report, Consistency or Initial
+   */
+  @visibility(Lifecycle.Read)
+  operationType?: Type;
+
+  /**
+   * The list of resources for which guest configuration assignment compliance is checked.
+   */
+  @OpenAPI.extension("x-ms-identifiers", #[])
+  resources?: AssignmentReportResource[];
+}
+
+/**
+ * The response model for the list of Automation operations.
+ */
+@pagedResult
+model OperationList {
+  /**
+   * List of Automation operations supported by the Automation resource provider.
+   */
+  @items
+  @OpenAPI.extension("x-ms-identifiers", #["name"])
+  value: Operation[];
+
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @nextLink
+  nextLink?: string;
```

Comment:
>string [](http://example.com/codeflow?start=13&length=6)

should nextLinks be format uri?

Can there be a typespec linter rule for this?

not really blocking but an open issue, so tagging as [ARMBlockingComment]